### ISSUE TITLE
ma.sh redirect stdout and stderr to file and don't display output on screen.

### ISSUE THREAD

**Issue by terrypacker** [infiniteautomation/ma-core-public#1323]
When you start Mango in an *nix environment the stdout and stderr are displayed in the shell, this can be annoying and we should make this optional.

**Comment by Puckfist** [comment]
Through the power of >/dev/null 2>&1 & it is optional!

**Comment by Puckfist** [comment]
Would an ext-enabled script work for you?

**Comment by terrypacker** [comment]
Yes I think that by default it not generate output and be optionally enabled by an extension.
